assignments/lambda-classes.js

```diff
@@ -1 +1,119 @@
 // CODE here for your Lambda Classes
+
+// base-class Person-----------------------------------------------------------
+class Person {
+    constructor(properties){
```

Comment:
Challenge for you. Learn about destrcuturing from MDN and try to implement it here.